lib/parser/sequentialStatementParser.ts

```diff
@@ -174,29 +174,40 @@ export class SequentialStatementParser extends ParserBase {
     this.expect(';');
     return assignment;
   }
-  parseExit(parent: O.OSequenceOfStatements | O.OIf, label?: OLexerToken) {
-    this.expect('exit');
-    const exitStatement = new O.OExit(parent, this.getToken().range.copyExtendEndOfLine());
-    exitStatement.label = label;
+  parseLoopWhen(statement: O.OExit|O.ONext) {
+
```

Comment:
whut is thi smae? parseLoopExitWhen or sth? This does not relate to loops really?

Reply:
exit and next statements are only allowed in loops and this parses the part where the loop label and a when part is allowed. Hence loop and when